index.html

```diff
@@ -391,14 +391,17 @@
 </div>
 <p>The <dfn>invoke idle callbacks algorithm</dfn>:</p>
 <ol>
+<li>Let <var>windows</var> be the <code>Window</code> object's list of <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing context</a>'s <code>WindowProxy</code> objects that have a non-empty <a>list of runnable idle callbacks</a> (<var>runlist</var>).</li>
```

Comment:
Hmm, I should have mentioned that this pull depends on https://github.com/w3c/requestidlecallback/pull/38 - related discussion in https://github.com/w3c/requestidlecallback/issues/25. Can you please take a look at that?